automation.iml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please, delete this file.

Reply:
I added *.iml to .ignore file. Please let me know if its wrong. In case, give me advice how can I delete this file because if I delete automation.iml , the src folder will  disappear.

Reply:
https://stackoverflow.com/questions/936249/how-to-stop-tracking-and-ignore-changes-to-a-file-in-git